apache-solr/dashboards/apache-solr-logs-overview.libsonnet

```diff
@@ -0,0 +1,34 @@
+local g = import 'github.com/grafana/grafonnet/gen/grafonnet-latest/main.libsonnet';
+local logsDashboard = import 'github.com/grafana/jsonnet-libs/logs-lib/logs/main.libsonnet';
+
+{
+  grafanaDashboards+::
+    if $._config.enableLokiLogs then
+      {
+        local apacheSolrLogsPanel =
+          logsDashboard.new(
+            'Apache Solr logs',
+            datasourceName='loki_datasource',
+            datasourceRegex='(?!grafanacloud.+usage-insights|grafanacloud.+alert-state-history).+',
+            filterSelector=$._config.filterSelector,
+            labels=['job', 'instance', 'level', 'solr_cluster', 'filename'],
```

Comment:
Since the order here determines the order on the dashboard, I think the cluster label should go before instance.

Reply:
updated